rfcs/2020-10-29-3736-remap-chain-parsing.md

```diff
@@ -78,11 +88,20 @@ if $match = matches(.message, /^Started (?P<method>[^\s]*) for (?P<remote_addr>[
 }
 ```
 
+The final statement must evaluate to a Boolean.
```

Comment:
What happens if it does not? Do we get a load-time error similar to https://github.com/timberio/vector/pull/4839 ?